### ISSUE TITLE
Exception: unknown datetime(6)

### ISSUE THREAD

**Issue by camillobruni** [philipsoutham/py-mysql2pgsql#83]
While running my conversion I got the following error

```
 153, in table_attributes
    columns.write('  %s,\n' % self.column_description(column))
  File "/usr/local/lib/python2.7/site-packages/mysql2pgsql/lib/postgres_writer.py", line 18, in column_description
    return '"%s" %s' % (column['name'], self.column_type_info(column))
  File "/usr/local/lib/python2.7/site-packages/mysql2pgsql/lib/postgres_writer.py", line 107, in column_type_info
    default, column_type = get_type(column)
  File "/usr/local/lib/python2.7/site-packages/mysql2pgsql/lib/postgres_writer.py", line 105, in get_type
    raise Exception('unknown %s' % column['type'])
Exception: unknown datetime(6)
```

I digged a bit deeper to find the offending table by printing the contents of `column`:

```
({
    'length': 6, 
    'table_name': u'django_migrations', 
    'name': u'applied', 
    'auto_increment': False, 
    'default': None, 
    'decimals': None, 
    'null': False, 
    'type': u'datetime(6)', 
    'primary_key': False
})
```

And the info from `mysql`:

```
mysql> DESCRIBE django_migrations;
+---------+--------------+------+-----+---------+----------------+
| Field   | Type         | Null | Key | Default | Extra          |
+---------+--------------+------+-----+---------+----------------+
| id      | int(11)      | NO   | PRI | NULL    | auto_increment |
| app     | varchar(255) | NO   |     | NULL    |                |
| name    | varchar(255) | NO   |     | NULL    |                |
| applied | datetime(6)  | NO   |     | NULL    |                |
+---------+--------------+------+-----+---------+----------------+
```

I don't have a clue what the length on a `datetime` field should do, but I can provide a fast fix by checking `column['type']` with a regex to avoid the above exception.

**Comment by camillobruni** [comment]
see pull-request #85

**Comment by kevgathuku** [comment]
Thanks @camillobruni 
I had the same issue and applying the fix in PR #85  solved it.

**Comment by kworr** [comment]
Fixed. Thank you!
